tests/yast2_gui/yast2_control_center.pm

```diff
@@ -24,12 +23,21 @@ sub search {
     # on openSUSE we have a Qt setup with keyboard shortcut
     if (check_var('DISTRI', 'opensuse')) {
         send_key 'alt-s';
+        type_string $name if $name;
```

Comment:
that does not make any sense. Please compare manually with https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/yast2_gui/yast2_control_center.pm and adjust accordingly. That type_string should be outside the if-part